gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/ParticipantReferenceUtil.java

```diff
@@ -4,13 +4,16 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
 import org.hl7.fhir.dstu3.model.Reference;
 import org.hl7.v3.II;
 import org.hl7.v3.RCMRMT030101UK04AgentRef;
 import org.hl7.v3.RCMRMT030101UK04EhrComposition;
 import org.hl7.v3.RCMRMT030101UK04Participant;
 import org.hl7.v3.RCMRMT030101UK04Participant2;
 
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
What is this all about?

Reply:
Has this been resolved? I'm also interested to know why it's here.